File: java/src/strategy/SimUDuckApp/Revision4/RubberDuck.java
package strategy.SimUDuckApp.Revision4;

public class RubberDuck extends Duck {
	public RubberDuck() {
		setFlyBehavior(new FlyNoWay());
		setQuackBehavior(new Squeak());
	}

	public void display() {
		System.out.println("I'm a rubber duckie");

	}

}

File: src/strategy/SimUDuckApp/Revision4/RubberDuck.kt
package strategy.SimUDuckApp.Revision4

class RubberDuck : Duck() {
    init {
        flyBehavior = FlyNoWay()
        quackBehavior = Squeak()
    }

    override fun display() {
        println("I'm a rubber duckie")
    }
}
